schema/fair.js

```diff
@@ -64,6 +65,15 @@ const FairType = new GraphQLObjectType({
         return null;
       },
     },
+    is_active: {
+      type: GraphQLBoolean,
+      description: 'Are we currently in the fair\'s active period?',
+      resolve: ({ autopublish_artworks_at, end_at }) => {
+        const start = moment.utc(autopublish_artworks_at).subtract(7, 'days');
+        const end = moment.utc(end_at).add(14, 'days');
```

Comment:
Yeah, we can use this in the interim but ultimately it's probably best to make the change in Gravity.